chore(docs): Release notes 2021_16 (#6689)
patternfly/patternfly-react: packages/react-docs/RELEASE-NOTES.md

- [@patternfly/react-table@4.50.1](https://www.npmjs.com/package/@patternfly/react-table/v/4.50.1)
- [@patternfly/react-tokens@4.33.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.33.1)
- [@patternfly/react-topology@4.28.1](https://www.npmjs.com/package/@patternfly/react-topology/v/4.28.1)
- [@patternfly/react-virtualized-extension@4.28.1](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.28.1)

### Components
- **Action list:**
  - Added support for classname ([#6647](https://github.com/patternfly/patternfly-react/pull/6647))
- **Alert:**
  -  Allowed all heading levels for title ([#6628](https://github.com/patternfly/patternfly-react/pull/6628))
- **Alert:**
  - Converted examples to functional with typescript ([#6634](https://github.com/patternfly/patternfly-react/pull/6634))
- **Card:**
  - Added disabled card and cleaned up props ([#6659](https://github.com/patternfly/patternfly-react/pull/6659))
  -  Updated card demos to use select instead of dropdown ([#6639](https://github.com/patternfly/patternfly-react/pull/6639))
- **Context selector:**

>   - Addedd support for focus trap to be made inactive ([#6638](https://github.com/patternfly/patternfly-react/pull/6638))

Reviewer: Removed extra "d"
```suggestion
  - Added support for focus trap to be made inactive ([#6638](https://github.com/patternfly/patternfly-react/pull/6638))
```

- **Date picker:**

>   - Updaetd to allow date picker users to open the calendar programatically ([#6632](https://github.com/patternfly/patternfly-react/pull/6632))

Reviewer: Spelling typos
```suggestion
  - Updated to allow date picker users to open the calendar programmatically ([#6632](https://github.com/patternfly/patternfly-react/pull/6632))
```

- **Drag drop:**
  - Fixed to not overwrite children class with droppable class ([#6672](https://github.com/patternfly/patternfly-react/pull/6672))
- **Dropdown:**
  - Added toggle variants ([#6651](https://github.com/patternfly/patternfly-react/pull/6651))
- **Dual list selector:**
  - Added composable tree example ([#6652](https://github.com/patternfly/patternfly-react/pull/6652))
- **Menu toggle:**
  - Added plain with text variant ([#6627](https://github.com/patternfly/patternfly-react/pull/6627))
- **Pagination:**
  - Updated `toggleTemplate` description to include `ofWord` prop ([#6622](https://github.com/patternfly/patternfly-react/pull/6622))
- **Popover:**
  - Added composable header and alert variants ([#6664](https://github.com/patternfly/patternfly-react/pull/6664))
  - Forwarded focus to close button with voice over click ([#6660](https://github.com/patternfly/patternfly-react/pull/6660))
- **Select:**
  - Added ability to persist filtered children state ([#6670](https://github.com/patternfly/patternfly-react/pull/6670))
  - Fixed no autocomplete for typeahead select ([#6623](https://github.com/patternfly/patternfly-react/pull/6623))
- **Slider:**

>    Fixed examples with custom steps and input ([#6662](https://github.com/patternfly/patternfly-react/pull/6662))

Reviewer: Missing bullet point
```suggestion
  - Fixed examples with custom steps and input ([#6662](https://github.com/patternfly/patternfly-react/pull/6662))
```
